### PR TITLE
Remove minified code blob

### LINKED ISSUE #26: What's with the obfuscated code?
https://github.com/papnkukn/qrcode-svg/blob/47d56ec28fcd0a3aabbb5f11a34167a02a80c54e/lib/qrcode.js#L88-L153

### PR DESCRIPTION
**Pull request by leMaik**
TL;DR If you want to get rid of the minified code blob, use `@lemaik/qrcode-svg`

---

Especially in a server application, running a blob of mangled JavaScript might scare some people (especially since recent npm package malware incidents), eg. this resolves #26 

Following my observations in https://github.com/papnkukn/qrcode-svg/issues/26#issuecomment-1159805781, I restored the proper code by performing these steps:

1. Pretty-print the code so that the following diffs make sense (either trust me or run prettier yourself to get the same result)
2. Replace the now pretty-printed blob with the latest version from jeromeetienne/jquery-qrcode, and pretty-printed it again.  
   Notably, this version contains https://github.com/kazuhikoarase/qrcode-generator/pull/41 and thus can determine the type automatically
3. Restore the utf8 fix that was previously in _this_ version, ie. apply https://github.com/davidshimjs/qrcodejs/commit/8247821f4a6336c8b7dd908a80f4e5c54e09fa1d
4. ~~Remove the type detection that got added in step 2, because that's [implemented already](https://github.com/papnkukn/qrcode-svg/blob/master/lib/qrcode.js#L276) here~~

### CODE REVIEW

**Comment by VanessaHenderson** [comment]
@papnkukn has this project been abandoned? Is there anychance you can merge this PR to remove the obfuscated code?
